Queue/Manager.php

```diff
@@ -196,7 +196,7 @@ public function getAllQueues()
     private function getVisitorIdFromRequest(Tracker\Request $request)
     {
         try {
-            $visitorId = $request->getVisitorId();
+            $visitorId = $request->getVisitorIdForThirdPartyCookie();
```

Comment:
could we make this backwards compatible with a check if that method exists? otherwise we'd need to increase the minimum required Matomo version just for this.

Reply:
good point, however, we already call it here
https://github.com/matomo-org/plugin-QueuedTracking/blob/50df72a1fb3fbf4ccdc53fb0cfd9deb1cfdc969a/Tracker/Handler.php#L47
so it should make no difference, I think

Reply:
Sorry I thought it was a new method that you added. Does it make sense to fall back to `getVisitorId()` if the method returned an empty result?

Reply:
Good idea, I have updated the code to call getVisitorId(true), which will never cause a DB connection, and make it always check for a forced userid